### PR TITLE
Upgrade Sequelize to v4, making sure BIGINT gets mapped to GraphQLString

### PR DESCRIPTION
**Pull request by stevehollaar**
Fixes https://github.com/mickhansen/graphql-sequelize/issues/539

If possible, can this also be cherry-picked to a `2.x` release?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
     "mysql": "^2.11.1",
     "pg": "^5.0.0",
     "pg-hstore": "^2.3.2",
-    "sequelize": "^3.24.6",
+    "sequelize": "^4.23.2",
```
Comment (mickhansen) [inline package.json]: need to test against ^3 aswell

Comment (RichAyotte) [reply package.json]: I've tested this patch with ^3 and ^4 and it works. Adding a dev dependencies for both versions is currently not possible with npm (https://github.com/npm/npm/issues/5499) However, it is with yarn. Would you consider using yarn?

Comment (RichAyotte) [reply package.json]: Also, would you accept the patch with the changes in package.json discarded for now?

Comment (mickhansen) [reply package.json]: > Would you consider using yarn?

Definitely consider yarn, use it a lot privately.

> Also, would you accept the patch with the changes in package.json discarded for now?

yes

**Pull request merged**